feature-libs/product-configurator/common/components/configure-cart-entry/configure-cart-entry.component.ts

```diff
@@ -53,13 +65,37 @@ export class ConfigureCartEntryComponent {
       throw new Error('No entryNumber present in entry');
     }
 
-    return this.cartEntry.orderCode
-      ? this.commonConfigUtilsService.getComposedOwnerId(
-          this.cartEntry.orderCode,
-          entryNumber
-        )
+    const code = this.getCode();
+    return code
+      ? this.commonConfigUtilsService.getComposedOwnerId(code, entryNumber)
       : entryNumber.toString();
   }
+  /**
+   * Returns a document code in case the entry is order or quote bound. In this case the code
+   * represents order or quote ID
+   * @returns Document code if order or quote bound, undefined in other cases
+   */
+  protected getCode(): string | undefined {
```

Comment:
I would suggest `getDocumentCode` instead of `getCode` or adjust method comment accordingly

Reply:
@ChristophHi, what do you think?

Reply:
Let's not do further changes here, as I anyhow plan to get rid of this method and simplify the component. See https://github.com/SAP/spartacus/pull/18024/ and refer to my discussion with Kris